src/Components/DownloadButtons/DownloadButtons.tsx

```diff
@@ -39,20 +29,20 @@ export const DownloadButtons = () => {
       },
     );
     saveAs(content, 'DIDsign-files.zip');
-  };
+  }
 
   const handleDownloadSign = async () => {
-    saveAs(signatureFile, 'signature.didsign');
-    if (isTimestamped) dispatch(updateDownloadStatus(true));
```

Comment:
Huh. It only set downloaded to true if it was timestamped? 😳

Reply:
Well, it only needed to track the downloaded status to prevent navigation while the timestamp has not been downloaded. This is not as bad as not resetting the downloaded status after the timestamp was added.

Reply:
Ahh right. The improved Navigation component makes that much clearer